Client/Application/Delegates/AppDelegate.swift

```diff
@@ -177,7 +177,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // the simulator via Xcode will count as a "crash" and lead to restore popups in the subsequent launch
         let crashedLastSession = !Preferences.AppState.backgroundedCleanly.value && AppConstants.buildChannel != .debug
         Preferences.AppState.backgroundedCleanly.value = false
-        browserViewController = BrowserViewController(profile: self.profile!, tabManager: self.tabManager, crashedLastSession: crashedLastSession)
+                    
+        guard let historyAPI = braveCore?.historyAPI, let bookmarkAPI = braveCore?.bookmarksAPI else {
+            return false
+        }
```

Comment:
We shouldn't return false here. Like I mentioned before, `braveCore` shouldn't be optional and should be initialized at definition

Reply:
Fix at https://github.com/brave/brave-ios/pull/4024/commits/91076d0df5e24f421b0ae0d49d4cbe1c0a6aabab